src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,310 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    CompoundInput,
+    DateInput,
+    // OptionPickerInput,
+    TextInput,
+    AutocompleteInput,
+    // IDGeneratorInput,
+    TermPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_DATE,
+  } = configContext.dataTypes;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return {
+    document: {
+      [config]: {
+        view: {
+          type: CompoundInput,
+          props: {
+            defaultChildSubpath: 'ns2:uoc_common',
+          },
+        },
+      },
+      ...extensions.core.fields,
+      'ns2:uoc_common': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/uoc',
+          },
+        },
+        referenceNumber: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.referenceNumber.name',
+                defaultMessage: 'Reference number',
+              },
+            }),
+            required: true,
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        methodList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          method: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.uoc_common.method.name',
+                  defaultMessage: 'Method',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: TermPickerInput,
+                props: {
+                  source: 'uocmethods',
+                },
+              },
+            },
+          },
+        },
+        title: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.title.name',
+                defaultMessage: 'Title',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        authorizedBy: {
+          [config]: {
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.authorizedBy.fullName',
+                defaultMessage: 'Authorized by',
+              },
```

Comment:
Don't need a fullName message if it's the same as the name message.

Reply:
Done